src/Compilers/CSharp/Portable/Binder/SwitchBinder_BindPatternSwitch.cs

```diff
@@ -27,25 +25,19 @@ internal PatternSwitchBinder(Binder next, SwitchStatementSyntax switchSyntax) :
         /// However, until we have edit-and-continue working, we continue using the old binder
         /// when we can.
         /// </summary>
-        private bool IsPatternSwitch
+        private bool UseV7SwitchBinder
         {
             get
             {
-                if (_isPatternSwitch == null)
```

Comment:
In the previous version this call was cached in a local.  Are there performance issues with respect to removing this caching?  Or was it unnecessary before?